mi/dataset/parser/utilities.py

```diff
@@ -67,7 +67,27 @@ def zulu_timestamp_to_ntp_time(zulu_timestamp_str):
     return float(ntplib.system_to_ntp_time(utc_time))
 
 
-def time_2000_to_ntp_time(time_2000):
+def julian_time_to_ntp(julian_timestamp_str):
+    """
+    Converts a julian formatted timestamp timestamp string to NTP time.
+    :param julian_timestamp_str: a julian formatted timestamp string
+    :return: NTP time in seconds
+    """
+
+    timestamp = datetime.strptime(julian_timestamp_str, "%Y%j")
+
+    return (timestamp - datetime(1900, 1, 1)).total_seconds()
+
+
+def seconds_1900_to_yyyy(year_in_yyyy):
+    """
+    :param year_in_yyyy: time string
```

Comment:
give an example string "2017" so it's clear - it's just the year that is used to calculate